Solar System/index.html

```diff
@@ -0,0 +1,139 @@
+<html>
+<head>
+    <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
+    <script src="https://cdn.rawgit.com/matthewbryancurtis/aframe-star-system-component/db4f1030/index.js"></script>
+</head>
+<body>
+    <a-scene background="color: #000000">
+        <a-assets>
+          <!--Planets-->
+          <img 
+            id="sun"
+            src="sun.jpg" />
+          <img 
+            id="mercury"
+            src="mercury.jpg" />
+          <img 
+            id="venus"
+            src="venus.jpg" />
+          <img 
+            id="earth"
+            src="earth.jpg" />
+          <img 
+            id="mars"
+            src="mars.jpg" />
+          <img 
+            id="jupiter"
+            src="jupiter.jpg" />
+          <img 
+            id="saturn"
+            src="saturn.jpg" />
+          <img 
+            id="uranus"
+            src="uranus.jpg" />
+          <img 
+            id="neptune"
+            src="neptune.jpg" />
+          <!--Sky-->
+          <audio
+            id="background" 
+            src="space.wav">
+          </audio>
+        </a-assets>
+        <a-entity star-system></a-entity>
+        <a-light
+          type="ambient"
+          position="-25 3 -10`"></a-light>
+        <a-sound 
+          src="#background"
+          autoplay="true"></a-sound>
+    </a-scene>
+    <script>
+    let distance = -20;
+    let planets = [
```

Comment:
Maybe deal with the sun separately

Reply:
While moving logic to a JavaScript file, a specific method for generating the sun was made.